### DISCUSSION TITLE
HMIP-RGBW add second channel

### DISCUSSION THREAD

**Discussion by semml0r** [SukramJ/aiohomematic#1384] [Ideas]
Dear all,  
as I run 4 x HMIP-RGBW, I wanted to ask if it is possible to include the second channel of the HMIP-RGBW in case it is set to run in 2-channel mode (I run only CCW LED stripes). Possibly also enabling to have the 4-channel mode would be really great.

In HA I can see the first channel (:1) is included and can be controlled, whereas the others cannot be controlled.

thanks a lot for this great intergation and best regards,
Simon

**Comment by SukramJ** [comment]
What happens if you activate the 2nd channel in HA?

**Reply by semml0r** [reply]
actually the second channel does not show up in HA. I took a screenshot of the first channel details, as attached. If I turn on the second channel via CCU3, it stays on and I cannot turn it off via HA.
<img width="549" alt="Bildschirmfoto 2024-01-15 um 15 31 10" src="https://github.com/danielperna84/hahomematic/assets/14950693/0cf6db42-88e3-4650-9196-f01fafa77a87">

**Reply by SukramJ** [reply]
Please post a screenshot of the device in HA.

**Reply by semml0r** [reply]
<img width="679" alt="Bildschirmfoto 2024-01-15 um 15 45 17" src="https://github.com/danielperna84/hahomematic/assets/14950693/406690f6-a954-419a-997d-0cda8694a3d1">

**Reply by SukramJ** [reply]
Which version of the integration are you running?
How is the device configured in CCU? (Mode)

What happens if you click on vch2? (Screenshot)

**Reply by semml0r** [reply]
- clicking on "vch2" brings up info that the entity is not available
- Homematic (IP) local: version 1.54.0 
- HMIP-RGBW in CCU configured mode: Tunable White

**Reply by SukramJ** [reply]
> clicking on "vch2" brings up info that the entity is not available

Are you able to press on delete, when the dialog is open?

Have you restarted HA after switching to tunable white?

**Reply by semml0r** [reply]
I apologize. This is entirely my bad. I can enable this channel in HA, it is disabled per default. Now I can also control the second channel from HA. Thanks a lot for the swift support!

**Reply by SukramJ** [reply]
No problem.
